llvm/test/tools/llvm-mca/RISCV/SiFive7/strided-load-x0.s

```diff
@@ -37,13 +37,13 @@ vle64.v v1, (a1)
 
 # CHECK:      Iterations:        1
 # CHECK-NEXT: Instructions:      26
-# CHECK-NEXT: Total Cycles:      234
+# CHECK-NEXT: Total Cycles:      211
```

Comment:
I would expect the fix for llvm-mca to be a NFC for its output, at least for all those scheduling model that are already existing and that by default have AcquireAtCycle = 0.

How do you explain this new values?

Reply:
I think `llvm-mca` has over estimated the cycles before. With this change, the cycles of RVV instrcctions should reduce by 1 cycle (for SiFive7VCQ), so here the total cycles reduce 23 cycles (there are 24 RVV instrcutions and extra 1 cycle for first RVV instruction?).